### ISSUE TITLE
problem about  the loss curves and the inpainting results  trained on  my own data

### ISSUE THREAD

**Issue by minushuang** [JiahuiYu/generative_inpainting#442]
Hi, Jiahui
I am grateful for you wonderful work and source code sharing. 
I'm training a model  on my own datasets. and according to your advise, I didn't change any hyper parameter in` inpaint.yml` except the `num_gpus_per_job` from `1` to ` 1,2,3` with multi GPU. besides,  I have about 70,000 images, and 10% of them are for validation and 90% for training.
the model have trained about 48h until now. but the loss curves and inpaint results seem to be a little strange. 
here are the loss curves
![image](https://user-images.githubusercontent.com/5811852/83933802-6f4b3700-a7de-11ea-9e84-be428f1a8a15.png)
![image](https://user-images.githubusercontent.com/5811852/83933813-8558f780-a7de-11ea-87f7-2c2a6ab7f818.png)
![image](https://user-images.githubusercontent.com/5811852/83933822-9c97e500-a7de-11ea-8430-7e45fa904aee.png)

and here are the val inpainting results during training
some like these
![image](https://user-images.githubusercontent.com/5811852/83933840-c4874880-a7de-11ea-9f62-db0abc8b82c5.png)
![image](https://user-images.githubusercontent.com/5811852/83933851-d79a1880-a7de-11ea-8a4a-cf6813939c5f.png)
![image](https://user-images.githubusercontent.com/5811852/83933860-e7196180-a7de-11ea-86a3-c567ef331eb1.png)

and some like these seems to be worse
![image](https://user-images.githubusercontent.com/5811852/83933880-1203b580-a7df-11ea-8b8a-71812b6396d4.png)
![image](https://user-images.githubusercontent.com/5811852/83933884-1d56e100-a7df-11ea-9e57-7ae760405a02.png)
![image](https://user-images.githubusercontent.com/5811852/83933914-3fe8fa00-a7df-11ea-8b43-f56cadf34537.png)

Q1. Is the loss curves normal? and what's meaning of these curves? the higher the value, the better the model ? or on the contrary? 

Q2. the inpainting results are not very good. some are blur, and some are with random strange colors. and can u give me some advice about improvement? more data? more training time ? loss ratio? any method you think can be used to improve the results.

Thank u again and any help will be appreciated

**Comment by JiahuiYu** [comment]
This model has failed so you can stop training. 

Q1: The curves look normal. GAN loss is hard to interpret but for AE loss the lower the better.

Q2: The training is already failed. Have you modified any of the training hyper-parameters? What's the batch size? I think if you exactly follow the hyper-parameters, it's likely to work on a 70K-image dataset.

**Comment by minushuang** [comment]
> This model has failed so you can stop training.
> 
> Q1: The curves look normal. GAN loss is hard to interpret but for AE loss the lower the better.
> 
> Q2: The training is already failed. Have you modified any of the training hyper-parameters? What's the batch size? I think if you exactly follow the hyper-parameters, it's likely to work on a 70K-image dataset.

No, I didn't chage any hyper-parameters except data path , and the batch-size is 16. and this is my `inpaint.yml`.
BTW, the mask I generated during training is a little different from the source code, the mask in my task is rectangle as the figures showed. Does that matter？
![image](https://user-images.githubusercontent.com/5811852/86099261-dad8a980-bae9-11ea-8c1f-172c3f41b7c1.png)
